apps/microtvm/reference-vm/base-box-tool.py

```diff
@@ -439,26 +473,27 @@ def parse_args():
         "--provider",
         choices=ALL_PROVIDERS,
         action="append",
-        help="Name of the provider or providers to act on; if not specified, act on all.",
+        required=True,
+        help="Name of the provider or providers to act on",
     )
 
-    parser.add_argument(
-        "platform",
-        help="Name of the platform VM to act on. Must be a sub-directory of this directory.",
-    )
+    # "test" has special options for different platforms, and "build", "release" might
+    # in the future, so we'll add the platform argument to each one individually.
+    platform_help_str = "Electronics platform to use (e.g. Arduino, Zephyr)"
```

Comment:
maybe change to "RTOS platforms to use..."?

Reply:
I don't think "RTOS platforms to use..." is appropriate, because Arduino is not an RTOS (nor are other potential platforms like CircuitPython). Arduino calls itself a "electronic prototyping platform", but that doesn't describe Zephyr well. 

I also don't like "platform to use", since "platform" could be confused with the other "microtvm-platform" parameter. Thoughts on other phrasings we could use?

Reply:
I think just `platform` makes more sense for Arduino/Zephyr. We can change `microtvm-platform` to `microtvm-target`. wdyt?

Reply:
We decided to rename `microtvm-platform` to `microtvm-device``